doc/make_qbk.py

```diff
@@ -198,7 +199,7 @@ def cs_to_quickbook(section):
 class_to_quickbook2("de9im_1_1static__mask", "de9im_static_mask")
 
 os.chdir("index")
-execfile("make_qbk.py")
+exec(compile(open("make_qbk.py", "rb").read(), "make_qbk.py", 'exec'))
```

Comment:
Wouldn't it be sufficient to call `exec(open("make_qbk.py").read())`?

As per https://docs.python.org/3.3/whatsnew/3.0.html#builtins
> Removed execfile(). Instead of execfile(fn) use exec(open(fn).read()).

Generally, catching up with Python 3 is a good idea, I think.

Reply:
I used the call used in `2to3` script; see https://github.com/python/cpython/blob/d4d60134b29290049e28df54f23493de4f1824b6/Lib/lib2to3/fixes/fix_execfile.py 
As far as I understand, that call has at least better error handling.

Reply:
Better errors handling is not entirely clear to me. Perhaps due to the fact `exec` and `compile` make two step action.
I've never bothered to use 2to3 really. Unless Python 2 support is desired, I'd stick to plain Python 3 API. Anyhow, I just wondered.